samples/net/sockets/echo_server/overlay-ws-console.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_SHELL_BACKEND_WEBSOCKET=y
+CONFIG_LOG_BACKEND_WS=y
+CONFIG_HTTP_SERVER_WEBSOCKET=y
+CONFIG_HTTP_SERVER=y
+CONFIG_EVENTFD=y
+CONFIG_POSIX_API=y
+CONFIG_FDTABLE=y
+CONFIG_POSIX_API=y
```

Comment:
POSIX API appears twice. I feel like there is some kind of Dos Equis humour to add here.

> I don't always use the websocket console, but when I do..

😅

Reply:
How did that meme become 18 years old all of a sudden 😬?

Reply:
Updated the prj.conf. Dunno why I did that, but I suppose it is always better to have things mentioned twice, just to be in safe side :grin: